doc/docs/master/deploy-manage/deploy/deploy-pachyderm-ide.md

```diff
@@ -0,0 +1,103 @@
+# Deploy the Pachyderm IDE
+
+!!! note
+    The Pachyderm IDE is an enterprise feature,
+    which is also available for testing during
+    the 14-day free trial.
+    Contact sales@pachyderm.com for more
+    information.
+
+The Pachyderm Integrated Development Environment (IDE) is
+an optional extension to Pachyderm clusters that provides a
+comprehensive environment for prototyping and deploying
+Pachyderm pipelines, as well as introspecting on data in the
+Pachyderm version control system. It combines together Jupyter,
+JupyterHub, and JupyterLab, which are familiar tools for many data
+scientists.
+
+JupyterLab is the newest Jupyter web interface that
+allows you to run your Jupyter notebooks in tabs and extend the
+tabs with custom applications. Our version of JupyterLab comes
```

Comment:
This isn't incorrect per se, but seems off in that it stresses the notion of tabs in JupyterLab, which is just kinda background music in IDEs in general

Reply:
rephrased